### ISSUE TITLE
Install fails on CentOS7

### ISSUE THREAD

**Issue by mikehwang** [aiortc/aioquic#47]
Here is the output when installing the aioquic project locally on a CentOS7 machine:

```
# pip3 install -e .                                                                                                                                                                                                
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.                                                                                                                            
Obtaining file:///home/aioquic                                                                                                                                                                                                                
Requirement already satisfied: cryptography>=2.5 in /usr/local/lib64/python3.6/site-packages (from aioquic==0.8.3)                                                                                                                            
Requirement already satisfied: pylsqpack<0.4.0,>=0.3.3 in /usr/local/lib64/python3.6/site-packages (from aioquic==0.8.3)                                                                                                                      
Requirement already satisfied: dataclasses in /usr/local/lib/python3.6/site-packages (from aioquic==0.8.3)                                                                                                                                    
Requirement already satisfied: six>=1.4.1 in /usr/local/lib/python3.6/site-packages (from cryptography>=2.5->aioquic==0.8.3)                                                                                                                  
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /usr/local/lib64/python3.6/site-packages (from cryptography>=2.5->aioquic==0.8.3)                                                                                                        
Requirement already satisfied: pycparser in /usr/local/lib/python3.6/site-packages (from cffi!=1.11.3,>=1.8->cryptography>=2.5->aioquic==0.8.3)                                                                                               
Installing collected packages: aioquic                                                                                                                                                                                                        
  Running setup.py develop for aioquic                                                                                                                                                                                                        
    Complete output from command /usr/bin/python3 -c "import setuptools, tokenize;__file__='/home/aioquic/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'ex
ec'))" develop --no-deps:                                                                                                                                                                                                                     
    running develop                                                                                                                                                                                                                           
    running egg_info                                                                                                                                                                                                                          
    writing src/aioquic.egg-info/PKG-INFO                                                                                                                                                                                                     
    writing dependency_links to src/aioquic.egg-info/dependency_links.txt                                                                                                                                                                     
    writing requirements to src/aioquic.egg-info/requires.txt                                                                                                                                                                                 
    writing top-level names to src/aioquic.egg-info/top_level.txt                                                                                                                                                                             
    reading manifest file 'src/aioquic.egg-info/SOURCES.txt'                                                                                                                                                                                  
    reading manifest template 'MANIFEST.in'                                                                                                                                                                                                   
    writing manifest file 'src/aioquic.egg-info/SOURCES.txt'                                                                                                                                                                                  
    running build_ext                                                                                                                                                                                                                         
    building 'aioquic._buffer' extension                                                                                                                                                                                                      
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -
fPIC -I/usr/include/python3.6m -c src/aioquic/_buffer.c -o build/temp.linux-x86_64-3.6/src/aioquic/_buffer.o                                                                                                                                  
    creating build/lib.linux-x86_64-3.6                                                                                                                                                                                                       
    creating build/lib.linux-x86_64-3.6/aioquic                                                                                                                                                                                               
    gcc -pthread -shared -Wl,-z,relro -g build/temp.linux-x86_64-3.6/src/aioquic/_buffer.o -L/usr/lib64 -lpython3.6m -o build/lib.linux-x86_64-3.6/aioquic/_buffer.cpython-36m-x86_64-linux-gnu.so                                            
    building 'aioquic._crypto' extension                                                                                                                                                                                                      
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -
fPIC -I/usr/include/python3.6m -c src/aioquic/_crypto.c -o build/temp.linux-x86_64-3.6/src/aioquic/_crypto.o                                                                                                                                  
    src/aioquic/_crypto.c: In function 'AEAD_decrypt':                                                                                                                                                                                        
    src/aioquic/_crypto.c:126:5: error: 'for' loop initial declarations are only allowed in C99 mode                                                                                                                                          
         for (int i = 0; i < 8; ++i) {                                                                                                                                                                                                        
         ^                                                                                                                                                                                                                                    
    src/aioquic/_crypto.c:126:5: note: use option -std=c99 or -std=gnu99 to compile your code                                                                                                                                                 
    src/aioquic/_crypto.c: In function 'AEAD_encrypt':                                                                                                                                                                                        
    src/aioquic/_crypto.c:168:5: error: 'for' loop initial declarations are only allowed in C99 mode                                                                                                                                          
         for (int i = 0; i < 8; ++i) {                                                                                                                                                                                                        
         ^                                                                                                                                                                                                                                    
    src/aioquic/_crypto.c: In function 'HeaderProtection_apply':                                                                                                                                                                              
    src/aioquic/_crypto.c:326:5: error: 'for' loop initial declarations are only allowed in C99 mode                                                                                                                                          
         for (int i = 0; i < pn_length; ++i) {                                                                                                                                                                                                
         ^                                                                                                                                                                                                                                    
    src/aioquic/_crypto.c: In function 'HeaderProtection_remove':                                                                                                                                                                             
    src/aioquic/_crypto.c:356:5: error: 'for' loop initial declarations are only allowed in C99 mode                                                                                                                                          
         for (int i = 0; i < pn_length; ++i) {                                                                                                                                                                                                
         ^                                                                                                                                                                                                                                   
    error: command 'gcc' failed with exit status 1                                                                                                                                                                                           
                                                                                                                                                                                                                                              
    ----------------------------------------
Command "/usr/bin/python3 -c "import setuptools, tokenize;__file__='/home/aioquic/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps
" failed with error code 1 in /home/aioquic/
```

I was able to get past the error by making the following modification (added the `extra_compile_args` bit) to the `setup.py`:

```
        setuptools.Extension("aioquic._buffer", sources=["src/aioquic/_buffer.c"], extra_compile_args=["-std=c99"]),
        setuptools.Extension(
            "aioquic._crypto", libraries=["crypto"], sources=["src/aioquic/_crypto.c"], extra_compile_args=["-std=c99"]
        ),
```

**Comment by sethmlarson** [comment]
Seems reasonable to add those flags, could you open a PR? :)

**Comment by jlaine** [comment]
Fixed in f1c915d9db6e1e63b02b5fbc58184d70f1baa51f
